applications/RomApplication/CMakeLists.txt

```diff
@@ -5,7 +5,41 @@ message("**** configuring KratosRomApplication ****")
 ################### PYBIND11
 include(pybind11Tools)
 
-include_directories( ${KRATOS_SOURCE_DIR}/kratos )
+add_definitions( -DEIGEN_MPL2_ONLY)

```

Comment:
Huuum, strong dependency

Reply:
My main problem is that it is hardcoded, today we discussed about this with @RiccardoRossi . For the moment no consensus how to replace spaces

Reply:
I am not an expert in this, maybe if @RiccardoRossi and/or @roigcarlo can give me a hand with this. I would appreciate it. I know we are making a strong dependency here. If you have any ideas, please let me know.